sdk/deviceregistry/arm-deviceregistry/src/api/deviceRegistryManagementContext.ts

```diff
@@ -19,11 +25,12 @@ export interface DeviceRegistryManagementClientOptionalParams extends ClientOpti
 /** Microsoft.DeviceRegistry Resource Provider management API. */
 export function createDeviceRegistryManagement(
   credential: TokenCredential,
+  subscriptionId: string,
   options: DeviceRegistryManagementClientOptionalParams = {},
 ): DeviceRegistryManagementContext {
-  const endpointUrl = options.endpoint ?? options.baseUrl ?? `https://management.azure.com`;
+  const endpointUrl = options.endpoint ?? options.baseUrl ?? "https://management.azure.com";
   const prefixFromOptions = options?.userAgentOptions?.userAgentPrefix;
-  const userAgentInfo = `azsdk-js-arm-deviceregistry/1.0.0-beta.2`;
+  const userAgentInfo = `azsdk-js-arm-deviceregistry/1.0.0-beta.1`;
```

Comment:
why this version is not updated?

Reply:
@wanlwanl could you work with ziwei to investigate why changelog tool didn't bump version?